From 9660b76325c841a5f5c7d5b2fb439a1fd64105c9 Mon Sep 17 00:00:00 2001
From: Xiangyu Chen <xiangyu.chen@windriver.com>
Date: Thu, 19 Jan 2023 17:07:37 +0800
Subject: [PATCH] messon.build: remove build path information to avoid yocto QA
 buildpath check warnings

libvirt debug trace contains source path, and some utils was located to
yocto sysroot folder, those full source path cause yocto QA buildpath check failed and report warnings as below:

WARNING: libvirt-8.1.0-r0 do_package_qa: QA Issue: File /usr/libexec/libvirt_lxc in package libvirt contains reference to TMPDIR
File /usr/lib64/libvirt.so.0.8001.0 in package libvirt contains reference to TMPDIR
File /usr/lib64/libvirt/lock-driver/lockd.so in package libvirt contains reference to TMPDIR
File /usr/lib64/libvirt/connection-driver/libvirt_driver_lxc.so in package libvirt contains reference to TMPDIR
File /usr/lib64/libvirt/connection-driver/libvirt_driver_storage.so in package libvirt contains reference to TMPDIR
File /usr/lib64/libvirt/connection-driver/libvirt_driver_network.so in package libvirt contains reference to TMPDIR
File /usr/lib64/libvirt/storage-backend/libvirt_storage_backend_fs.so in package libvirt contains reference to TMPDIR
File /usr/lib64/libvirt/storage-backend/libvirt_storage_backend_disk.so in package libvirt contains reference to TMPDIR
File /usr/bin/virt-ssh-helper in package libvirt contains reference to TMPDIR [buildpaths]


Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Xiangyu Chen <xiangyu.chen@windriver.com>
---
 meson.build | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

Index: libvirt-v11.4.0+git/meson.build
===================================================================
--- libvirt-v11.4.0+git.orig/meson.build
+++ libvirt-v11.4.0+git/meson.build
@@ -38,8 +38,8 @@
 conf = configuration_data()
 
 conf.set('_GNU_SOURCE', 1)
-conf.set_quoted('abs_top_builddir', meson.project_build_root())
-conf.set_quoted('abs_top_srcdir', meson.project_source_root())
+conf.set_quoted('abs_top_builddir', ' ')
+conf.set_quoted('abs_top_srcdir', ' ')
 conf.set_quoted('PACKAGE', meson.project_name())
 conf.set_quoted('PACKAGE_NAME', meson.project_name())
 conf.set_quoted('PACKAGE_VERSION', meson.project_version())
